Removed unwanted end-of-line conversions

rajdakin opened the pull request:
Removes every EOL conversion for every PNG, JPG and GIF files that Git may attempt (note: there is currently 26 PNG, 8 JPG and one GIF that Git warns about).

Butt4cak3 commented:
I don't know which warnings you're talking about. Please explain:

1. What version of Git are you running on which operating system?
2. When do the warnings appear? Which commands do you run to get the warnings?

rajdakin commented:
Hello.
I'm running on Linux Mint 18.3 Sylvia (based on Ubuntu Xenial).
On my PS1, I run ``` `__git_ps1 " (at %s) by "``( git reflog --format=full | head -4 | tail -1 | cut -d" " -f2- | cut -d"<" -f1 | sed "s/ $//" ) 2>/dev/null` ``` with Git 2.7.4.
The warning appears at the moment I go into the repo.
All this is in the command line; the terminal is the MATE terminal version 1.18.2.

The exact warning is:
```
warning: CRLF will be replaced by LF in chapters/FFT/res/FT_example.png.
The file will have its original line endings in your working directory.
```
And the `git status` command returned me:
```
	modified:         chapters/FFT/res/FT_example.png
	modified:         chapters/FFT/res/butterfly_diagram.png
	modified:         chapters/FFT/res/fft.png
	modified:         chapters/FFT/res/radix-2screen.jpg
	modified:         chapters/FFT/res/radix-2screen_positive.jpg
	modified:         chapters/FFT/res/radix-8screen.jpg
	modified:         chapters/FFT/res/sinusoid.png
	modified:         chapters/data_compression/huffman/res/huffman_tree.png
	modified:         chapters/differential_equations/euler/res/error.png
	modified:         chapters/differential_equations/euler/res/instability.png
	modified:         chapters/euclidean_algorithm/res/modulus.png
	modified:         chapters/euclidean_algorithm/res/subtraction.png
	modified:         chapters/monte_carlo/res/13311.png
	modified:         chapters/monte_carlo/res/195583.png
	modified:         chapters/monte_carlo/res/31.png
	modified:         chapters/monte_carlo/res/monte_carlo.gif
	modified:         chapters/monte_carlo/res/square_circle.png
	modified:         chapters/physics_solvers/verlet/code/labview/verlet_labview.png
	modified:         chapters/physics_solvers/verlet/code/scratch/verlet_scratch.png
	modified:         chapters/principles_of_code/building_blocks/nand.jpg
	modified:         chapters/principles_of_code/building_blocks/nor.jpg
	modified:         chapters/principles_of_code/building_blocks/not.jpg
	modified:         chapters/principles_of_code/building_blocks/or.jpg
	modified:         chapters/principles_of_code/building_blocks/xor.jpg
	modified:         chapters/principles_of_code/clone.png
	modified:         chapters/principles_of_code/fork.png
	modified:         chapters/principles_of_code/notation/cplot.png
	modified:         chapters/taylor/function_sum.png
	modified:         chapters/tree_traversal/code/scratch/scratch_tree.png
	modified:         chapters/tree_traversal/res/3-tree.png
	modified:         chapters/tree_traversal/res/BFS_simple.png
	modified:         chapters/tree_traversal/res/DFS_in.png
	modified:         chapters/tree_traversal/res/DFS_post.png
	modified:         chapters/tree_traversal/res/DFS_pre.png
	modified:         chapters/tree_traversal/res/binary_tree.png
```
before modifying the `.gitattribute`, after which it told me that only `.gitattribute` was modified.

Gustorn commented:
This has been [fixed](https://github.com/git/git/blob/master/Documentation/RelNotes/2.10.0.txt) in Git 2.10.0 which is over 1.5 years old at this point. I think it's reasonable to expect that people use a _somewhat_ up to date version of git.

Butt4cak3 commented:
Pretty much what @Gustorn said. Try to update Git to a more recent version, `git reset --hard` to  clean your working directory (make sure everything else is commited) and see if the problem persists.

rajdakin commented:
Unfortunately, Linux Mint has almost no up-to-date softwares from their repos, and I'll need sources or PPAs to have it up to date...
Anyways, I'll try to update my Git version (using the Git repo) and close this pull request.
